src/basic-languages/python/python.ts

```diff
@@ -250,16 +250,32 @@ export const language = <languages.IMonarchLanguage>{
 		// Recognize strings, including those broken across lines with \ (but not without)
 		strings: [
 			[/'$/, 'string.escape', '@popall'],
-			[/f'{1,3}/, 'string.escape', '@fStringBody'],
+			[/f'{3}/, 'string.escape', '@fStringBody3'],
+			[/f'{1}/, 'string.escape', '@fStringBody1'],
 			[/'/, 'string.escape', '@stringBody'],
 			[/"$/, 'string.escape', '@popall'],
-			[/f"{1,3}/, 'string.escape', '@fDblStringBody'],
+			[/f"{3}/, 'string.escape', '@fDblStringBody3'],
+			[/f"{1}/, 'string.escape', '@fDblStringBody1'],
 			[/"/, 'string.escape', '@dblStringBody']
 		],
-		fStringBody: [
+		fStringBody3: [
+			[/[^\\'\{\}]+/, 'string'],
```

Comment:
I have several questions about the PR which I would like to ask to better understand this PR.

How come here on line 262 it's written 

`[/[^\\'\{\}]+/, 'string']`

But on line 273 we have:

			[/[^\\'\{\}]+$/, 'string', '@popall'],
			[/[^\\'\{\}]+/, 'string'],

For the other case? It seems like this code is not the exact duplicate of the other code, as it does not contain the third value '@popall'.

Also would you know what popall here refers to? I am trying to understand.